### ISSUE TITLE
v3: "swagger:example" is not being honored when set to "false" on API/Attribute level

### ISSUE THREAD

**Issue by nickchen** [goadesign/goa#2206]
Using the following

```
package design

import (
	. "goa.design/goa/v3/dsl"
)

var _ = API("Pet", func() {
	Meta("swagger:example", "false")

	Title("Pet demo project to test out goa capabilities")
	HTTP(func() {
		Consumes("application/json")
		Produces("application/json")
	})
	Server("app", func() {
		Host("localhost", func() {
			URI("https://localhost:8080")
		})
	})
})

var _ = Service("PetStore", func() {

	HTTP(func() {
		Path("/api")
	})
	Method("get", func() {
		Description("Gets a single Pet.")
		Payload(func() {
			Attribute("pet_id", String, "id of the strategy")
			Required("pet_id")
		})

		Result(Pet)
		Error("not_found", String, "Pet not found")
		HTTP(func() {
			GET("/pets/{pet_id}")
			Response(StatusOK, func() {
				// this sets the name for the reference, so we don't create a duplicated object
				Body(Pet)
			})
			Response("not_found", StatusNotFound)
		})
	})
	Method("list", func() {
		Description("Gets a list of all Pets.")
		Result(Pets)
		HTTP(func() {
			GET("/pets")
			Response(StatusOK, func() {
				Body(Pets)
			})
		})
	})
})

var PetType = Type("PetType", String, func() {
	Enum("FOUR_LEG", "TWO_LEG")
})

var PetAttribute = Type("PetAttribute", String, func() {
	Enum("MEAT", "VEGGIE", "LIKES_SUN", "DOESNOT_LIKE_SUN")
})

var PetAttributeContainer = Type("PetMode", func() {
	Attribute("values", String)
	Attribute("attributes", ArrayOf(PetAttribute), func() {
		Meta("swagger:example", "false")
		Example(func() {
			Value([]string{"MEAT"})
		})
	})
})

var Pet = Type("Pet", func() {
	Attribute("name", String, `name of the pet`, func() {
		MaxLength(24)
	})
	Attribute("type", PetType)
	Attribute("attributes", PetAttributeContainer)

})
var Pets = Type("Pets", ArrayOf(Pet))
```

Expect examples not be generated, but still generates examples.

**Comment by nitinmohan87** [comment]
Thanks for the report!

I think we need to rename `swagger:example` to be more generic because examples are also used when generating CLI.

**Comment by ilmaruk** [comment]
Has this been resolved? I'm having the same issue with v3.0.6. Thanks.

**Comment by raphael** [comment]
Thank you for fixing the issue @ilmaruk !
